contracts/Commitment/Commitment.sol

```diff
@@ -565,4 +540,15 @@ contract Commitment is
         }
         return investorNmk;
     }
+
+    // calculates investor's and platform operator's neumarks from total reward
+    function calculateNeumarkDistribtion(uint256 rewardNmk)
+        private
+        returns (uint256 platformNmk, uint256 investorNmk)
+    {
+        // round down - platform may get 1 wei less than investor
```

Comment:
Do you have a test case where it actually happens?

Reply:
added in #5c7fbb1